### PR TITLE
Put CVE-2018-16301 right

### PR DESCRIPTION
**Pull request by infrastation**
See the commit message for clarifications.

### CODE REVIEW

File: 2018/16xxx/CVE-2018-16301.json [hunk 1]
```diff
@@ -5,13 +5,72 @@
     "CVE_data_meta": {
```
Comment (cve-team) [inline 2018/16xxx/CVE-2018-16301.json:5]: CNA(s) of user (Tcpdump) do not match owning CNA.

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by infrastation** [comment]
It looks like this request will require manual merging. The CVE was assigned by MITRE before The Tcpdump Group became a CNA, so MITRE indeed is the owning CNA, yet this entry requires these corrections.

**Comment by anthnysingleton** [comment]
Correct we have uploaded and process this record change request.

**Comment by infrastation** [comment]
Thank you, I see the update in git. Does the [web version](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-16301) require a manual update to synchronize?

**Comment by anthnysingleton** [comment]
The record will be updated on the sites in the next few hours. No additional work needed.

**Comment by infrastation** [comment]
That's grand then, thanks again.

**Pull request closed without merging**